lib/ibancom/resources/iban/validation.rb

```diff
@@ -29,7 +29,10 @@ def supported_schemes
 
         def supported_scheme?(code)
           scheme = supported_schemes.find { |supported| supported.code == code }
-          scheme&.supported?
+
+          return false if scheme.nil?
+
+          scheme.supported?
```

Comment:
🤔 
```ruby
scheme&.supported? == nil.supported? == false
```
I still do not understand why we need to check nullability in an explicit way